### PR TITLE
feat:add semantic html lab

### LINKED ISSUE #385: chore: add semantic HTML only lab to frontend cert
The semantic HTML section will have a few labs for campers to do.
Labs are defined as short projects campers build on their own. It will consist of a blank editor and user stories.
Here is an example for the format:
https://www.freecodecamp.org/learn/the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project

Campers will have learned about semantic HTML in depth with the following project
https://github.com/freeCodeCamp/CurriculumExpansion/pull/384/files

So we need a lab project here that covers the following:

- header
- nav
- main
- sections and articles

## Ideas for lab projects

- magazine page
- events page
- book listing page

## length of project

This lab should be of similar length of the blog page tutorial that comes before it.
So probably 8-10 user stories

## note about the HTML boilerplate

At this point in the cert, campers will have created a few labs and worked on a few workshops.
So we no longer need to force them to write out the HTML boilerplate each time.
So we don't need to create user stories for the HTML boilerplate anymore. 

## Note about adding new images

For adding new images, make sure they are free to use. You will also need to be add that image to our CDN repo here
https://github.com/freeCodeCamp/cdn

Once the new image has been added to the CDN repo, then you will need to reference that cdn link in your project.

If you are not sure about if an image can be used, reach out on staff chat. 

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/html-projects` directory
- [ ] add your completed project in an `index.html` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by larymak**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #385

screenshot-127 0 0 1_5500-2024 07 28-10_29_39

### CODE REVIEW

File: frontend-cert/html-projects/event-page/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+1. You should have a `header` element.
+1. Inside the `header` element, you should have a `h1` element that contains the text `Event Hub`, and a `nav` element.
+1. Inside the `nav` element, you should have an unordered list of two items containing links to different sections of the page. The first item should have the text `Upcoming Events`, and the second item should have the text `Past Events`.
+1. Each link should be represented by an `a` element with an `href` attribute that links to the corresponding section of the page, `#upcoming` and `#past` respectively.
+1. You should have a `main` element that contains the different sections of the page.
+1. Inside the `main` element, you should have atleast two `section` elements.
```
Comment (Dario-DC) [inline frontend-cert/html-projects/event-page/user-stories.md:6]:
```suggestion
1. Inside the `main` element, you should have at least two `section` elements.
```

**Comment by jdwilkin4** [comment]
Hey @larymak ! 

It looks like you still have the CSS grid project in here.
Can you update your PR to remove that please?

An then once the merge conflicts are resolved, then should be good to go.

**Comment by moT01** [comment]
I see a lot of leftovers of an attempted merge conflict fix. You should remove those @larymak. Let me know if you need any help.

**Comment by larymak** [comment]
> I see a lot of leftovers of an attempted merge conflict fix. You should remove those @larymak. Let me know if you need any help.

@moT01 yeah I would appreciate some help with this kindly

**Comment by moT01** [comment]
I see two folders, a `css-grid` folder, and an `event-page` folder. We can just delete the css-grid folder correct? The stuff we want is in the event-page folder?

**Comment by larymak** [comment]
> I see two folders, a `css-grid` folder, and an `event-page` folder. We can just delete the css-grid folder correct? The stuff we want is in the event-page folder?

Yeah, that's correct

**Comment by moT01** [comment]
Do you want to try deleting it? Let me know if you need help.

**Comment by larymak** [comment]
> Do you want to try deleting it? Let me know if you need help.

Done, The CSS folder has been deleted.

**Comment by Dario-DC** [comment]
I see that the grid project has been deleted, so merging this PR would remove it from the main branch. 
This PR should not touch the files in the /css-grid folder at all. Is that right? @moT01 @larymak

**Review by jdwilkin4 — APPROVED** [review]
I pushed up a commit to add the css grid project back in because it was originally going to be deleted all together. 

The diff changes you see, are to remove the extra unnecessary blank lines at the bottom of the html file and user stories files. 

Otherwise, I think this project is good to go in. 

Don't forget to open up a PR in the cdn repo for the images you added.
Also, it helps if you mention where these images came from in that CDN repo PR

**Pull request merged**